src/proof/proof_rule.h

```diff
@@ -1779,6 +1779,46 @@ enum class PfRule : uint32_t
   // over a Cell (in variables x1...xn). It derives that A evaluates to false
   // over the Cell. In the actual algorithm, it means that xn can not be in the
   // topmost interval of the Cell.
+  /**
```

Comment:
Should this subsume the previous documentation?  Delete the old documentation above this line?